src/engine/SCons/Tool/MSCommon/vc.py

```diff
@@ -296,9 +296,9 @@ def find_vc_pdir_vswhere(msvc_version):
     vswhere_cmd = [vswhere_path, '-products', '*', '-version', msvc_version, '-property', 'installationPath']
 
     if os.path.exists(vswhere_path):
-         #TODO PY27 cannot use Popen as context manager
-         # try putting it back to the old way for now
-         sp = subprocess.Popen(vswhere_cmd,
+        #TODO PY27 cannot use Popen as context manager
```

Comment:
Can you just have if here with context manger when py3?

Reply:
Sorry, don't follow? did you mean leave the Py3 code in place commented out?  I'm exploring if there's a wrapper class we could make for Popen that would bring this functionality in, doesn't look too hard. But I think that's a separate experiment, _subproc function is marked in-progress anyway and could become the wrapper;